ui/src/app/pages/admin/admin-egress-audit.tsx

```diff
@@ -117,36 +117,46 @@ export const AdminEgressAudit = (props: WithSpinnerOverlayProps) => {
   const event = egressDetails.egressEvent;
   const eventChanged =
     !!pendingUpdateEvent && !fp.equals(pendingUpdateEvent, event);
-  const [username] = event.sourceUserEmail.split('@');
+
+  const {
+    sourceUserEmail,
+    sourceWorkspaceNamespace,
+    sourceGoogleProject,
+    status,
+    egressEventId,
+    egressMegabytes,
+    egressWindowSeconds,
+    timeWindowStartEpochMillis,
+    timeWindowEndEpochMillis,
+  } = event;
+
+  const [username] = sourceUserEmail.split('@');
+
   return (
     <div style={{ padding: '0 20px' }}>
-      <h2>Egress event {event.egressEventId}</h2>
+      <h2>Egress event {egressEventId}</h2>
       <div style={{ display: 'table', marginBottom: '15px' }}>
-        <DetailRow label='Detection time'>
-          {new Date(event.creationTime).toLocaleString()}
```

Comment:
This was confusing when I investigated an egress event, because this does not refer to the egress window - rather it's some time after the end of that window.  (In this particular case, 5 min)